crates/ironrdp-displaycontrol/src/client.rs

```diff
@@ -1,14 +1,47 @@
 use crate::{
-    pdu::{DisplayControlMonitorLayout, DisplayControlPdu, MonitorLayoutEntry},
+    pdu::{DisplayControlCapabilities, DisplayControlPdu},
     CHANNEL_NAME,
 };
 use ironrdp_dvc::{encode_dvc_messages, DvcClientProcessor, DvcMessage, DvcProcessor};
-use ironrdp_pdu::PduResult;
+use ironrdp_pdu::{cursor::ReadCursor, PduDecode, PduResult};
 use ironrdp_svc::{impl_as_any, ChannelFlags, SvcMessage};
 use tracing::debug;
 
 /// A client for the Display Control Virtual Channel.
-pub struct DisplayControlClient {}
+pub struct DisplayControlClient {
+    /// A callback that will be called when capabilities are received from the server.
+    /// If no callback is set, a default (inert) callback will be used.
```

Comment:
I think this comment needs to be adjusted after previous refactoring, as there is no way to create the client now without explicitly setting the callback